shared/metrics/__init__.py

```diff
@@ -1,5 +1,9 @@
+import logging
+
 from prometheus_client import Counter, Histogram, Summary, start_http_server
 
+log = logging.getLogger(__name__)
```

Comment:
@michelletran-codecov 
The new logger `_name_` would probably be `shared.shared.metrics` so do you think it would be harder to trace back to where the error occured knowing the `_name` of the Counter only?